js/d3-tip.js

```diff
@@ -0,0 +1,352 @@
+/**
```

Comment:
This file could be included from unpkg as well as `<script src="https://unpkg.com/d3-tip@0.7.1"></script>`